modules/@apostrophecms/modal/ui/apos/mixins/AposModalTabsMixin.js

```diff
@@ -46,10 +46,13 @@ export default {
       const tabs = [];
       for (const key in this.groups) {
         if (key !== 'utility') {
+          const conditionalFields = this.conditionalFields('other');
+          const fields = this.groups[key].fields;
           tabs.push({
             name: key,
             label: this.groups[key].label,
-            fields: this.groups[key].fields
+            fields,
+            isVisible: fields.filter(field => conditionalFields[field] !== false).length > 0
```

Comment:
You could use the `some` method here to directly get a boolean, no need to check the length.
More readable I think:
```javascript
isVisible: fields.some(field => conditionalFields[field] !== false)
```